jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>"Highly variable gene selection can ensure that scGen only focuses on the most varying and therefore potentially biologically relevant genes." BAD this is so bad, really bad, honestly terrible advice. You can use it if you want but I wouldn't even recommend hrnngg</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>